### ISSUE TITLE
It should show get pid but get username instead

### ISSUE THREAD

**Issue by chen0423** [neekey/ps#13]
line 221: 
` var pid = ( d.PID && d.PID[ 0 ] ) || ( d.ProcessId && d.ProcessId[ 0 ] ) || undefined;`
should be:
` var pid = ( d.PID && d.PID[ 1 ] ) || ( d.ProcessId && d.ProcessId[ 1 ] ) || undefined;`
In order to get pid instead of username.

Not sure if this issue happen because of system dependency(I use Mac OS X and Ubuntu MATE).
Or should I send a PR directly?

**Comment by neekey** [comment]
@chen0423 the result data is parsed from Table parse, and every field is an array, and in most of the cases, the array only contains one element, like:

```js
pid: [ 1234 ]
```

but fields like `COMMAND` will has more than one element, like:

```js
COMMAND: [ 'node', '--debug' ]
```

So this it is why I use the index 0 for most fields.

Anyway, could you post your code that causes you any problem with this?

**Comment by napalm272** [comment]
I can verify that this happens. I'm just using the example code
```
var ps = require('ps-node');
 
// A simple pid lookup 
ps.lookup({
    command: 'node',
    arguments: '--debug',
    }, function(err, resultList ) {
    if (err) {
        throw new Error( err );
    }
 
    resultList.forEach(function( process ){
        if( process ){
 
            console.log( 'PID: %s, COMMAND: %s, ARGUMENTS: %s', process.pid, process.command, process.arguments );
        }
    });
});
```

I'm running this on a raspberry pi (kernel=3.18) and node 0.10.

**Comment by neekey** [comment]
@chen0423 @napalm272 this bug should be fixed in `0.1.3` https://github.com/neekey/ps/pull/28
